pushy/src/main/java/com/turo/pushy/apns/util/ApnsPayloadBuilder.java

```diff
@@ -530,6 +535,58 @@ public ApnsPayloadBuilder setThreadId(final String threadId) {
         return this;
     }
 
+    /**
+     * <p>Sets the summary argument for this notification. The summary argument is:</p>
+     *
+     * <blockquote>The string the notification adds to the category’s summary format string.</blockquote>
+     *
+     * <p>For example when defining an UNNotificationCategory, passing a format string like
+     * "%u more messages from %@" to the categorySummaryFormat argument, will produce
+     * "x more messages from {summaryArgument}.</p>
+     *
+     * <p>By default, it says “x more notifications”, no summary argument is included.</p>
+     *
+     * @param summaryArgument the summary argument for this notification
+     *
+     * @return a reference to this payload builder
+     *
+     * @see <a href=
+     *      "https://developer.apple.com/documentation/usernotifications/unnotificationcontent">
+     *      UNNotificationContent</a>
+     *
+     * @since 0.13.6
+     */
+    public ApnsPayloadBuilder setSummaryArgument(final String summaryArgument) {
+        this.summaryArgument = summaryArgument;
+        return this;
+    }
+
+    /**
+     * <p>Sets the summary argument count for this notification. The summary argument count is:</p>
+     *
+     * <blockquote>The number of items the notification adds to the category’s summary format string.</blockquote>
+     *
+     * <p>For example when defining an UNNotificationCategory, passing a format string like
+     * like "%u more podcasts" to the categorySummaryFormat argument, will produce "{argumentSummaryCount} more podcasts"
+     * </p>
+     *
+     * <p>By default, summary argument count is 1 and it can not be 0.</p>
```

Comment:
Minor phrasing suggestion:

> By default, notifications count as a single "item" in a group of notifications, and so the default summary argument count is 1 (even if no summary argument count is specified in the notification payload). If specified, summary argument counts must be positive.

Relatedly, should we actually enforce that restriction in this setter? We could throw an `IllegalArgumentException` if the count is non-null and less than 1.

Reply:
Throwing the exception in cd6eb42.